index.d.ts

```diff
@@ -5,6 +5,12 @@ import {
   LightMyRequestResponse,
 } from 'fastify'
 
+declare module 'fastify' {
+	interface FastifyInstance {
+		restart: () => Promise<void>
```

Comment:
Nitpick: we use 2 spaces as spacing.

Reply:
Sorry bad setting on IDE... now is set to 2 spaces